### PR TITLE
Added helm changes in deployment docs

### PR DESCRIPTION
**Pull request by evalsocket**
- Update helm registry in docs
- Added docs in eks deployment for flytescheduler and aws native scheduler

Screenshot 2021-12-08 at 7 03 32 PM

### CODE REVIEW

File: rsts/deployment/aws/manual.rst [hunk 1]
```diff
@@ -407,11 +407,33 @@ Time for Helm
 
 Installing Flyte
 -----------------
-#. Clone the Flyte repo
 
-.. code-block:: bash
+#. Add the flyte chart repo to helm
+
+  .. code-block::
```
Comment (EngHabu) [inline rsts/deployment/aws/manual.rst:413]: In the rendered view this like like a quoted string... I think it has an extra space in the beginning of the line... Maybe this line or maybe the next line is the problem

File: rsts/deployment/aws/manual.rst [hunk 2]
```diff
@@ -407,11 +407,33 @@ Time for Helm
 
 Installing Flyte
 -----------------
-#. Clone the Flyte repo
 
-.. code-block:: bash
+#. Add the flyte chart repo to helm
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:411]:
```suggestion
#. Add Flyte chart repo to Helm
```

File: rsts/deployment/aws/manual.rst [hunk 3]
```diff
@@ -407,11 +407,33 @@ Time for Helm
...
+
+  .. code-block::
+
+     helm repo add flyteorg https://flyteorg.github.io/flyte
+
+
+#. Download eks values for helm
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:418]:
```suggestion
2. Download eks values for Helm
```

File: rsts/deployment/aws/manual.rst [hunk 4]
```diff
@@ -407,11 +407,33 @@ Time for Helm
...
+
+   .. tabbed:: Flyte Native scheduler
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:420]:
```suggestion
   .. tabbed:: Flyte Native Scheduler
```

File: rsts/deployment/aws/manual.rst [hunk 5]
```diff
@@ -407,11 +407,33 @@ Time for Helm
...
+
+      * Download eks helm values, By default it enable the flyte native scheduler:
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:422]:
```suggestion
      * Download eks Helm values (it enables Flyte native scheduler by default)
```

File: rsts/deployment/aws/manual.rst [hunk 6]
```diff
@@ -407,11 +407,33 @@ Time for Helm
...
+
+        .. code-block:: bash
+
+           curl -sL https://raw.githubusercontent.com/flyteorg/flyte/master/charts/flyte-core/values-eks.yaml
+
+   .. tabbed:: AWS scheduler
+
+      * Download eks helm values for aws scheduler
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:430]:
```suggestion
      * Download eks helm values for AWS scheduler
```

File: rsts/deployment/aws/manual.rst [hunk 7]
```diff
@@ -407,11 +407,33 @@ Time for Helm
...
+
+        .. code-block:: bash
+
+           curl -sL https://raw.githubusercontent.com/flyteorg/flyte/master/charts/flyte-core/values-eks.yaml
+           curl -sL https://raw.githubusercontent.com/flyteorg/flyte/master/charts/flyte-core/values-eks-override.yaml
 
-   git clone https://github.com/flyteorg/flyte
 
 #. Update values
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:438]:
```suggestion
3. Update values in the YAML file
```

File: rsts/deployment/aws/manual.rst [hunk 8]
```diff
@@ -446,19 +468,24 @@ Search and replace the following
      - ARN of the self-signed (or official) certificate
      - ``arn:aws:acm:us-east-2:173113148371:certificate/763d12d5-490d-4e1e-a4cc-4b28d143c2b4``
 
-#. Update helm dependencies
 
-.. code-block:: bash
+#. Install Flyte
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:472]:
```suggestion
4. Install Flyte
```

File: rsts/deployment/aws/manual.rst [hunk 9]
```diff
@@ -446,19 +468,24 @@ Search and replace the following
...
 
-   helm dep update
+   .. tabbed:: Flyte Native scheduler
 
+      * Install Flyte with flyte native scheduler:
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:476]:
```suggestion
      * Install Flyte with Flyte native scheduler
```

File: rsts/deployment/aws/manual.rst [hunk 10]
```diff
@@ -446,19 +468,24 @@ Search and replace the following
...
 
-#. Install Flyte
+        .. code-block:: bash
 
-.. code-block:: bash
+           helm install -n flyte -f values-eks.yaml --create-namespace flyte flyteorg/flyte-core
+
+   .. tabbed:: AWS scheduler
+
+      * Install Flyte with flyte aws scheduler
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:484]:
```suggestion
      * Install Flyte with Flyte AWS scheduler
```

File: rsts/deployment/aws/manual.rst [hunk 11]
```diff
@@ -446,19 +468,24 @@ Search and replace the following
...
+
+        .. code-block:: bash
 
-   cd helm
-   helm install -n flyte -f values-eks.yaml --create-namespace flyte .
+           helm install -n flyte -f values-eks.yaml -f values-eks-override --create-namespace flyte flyteorg/flyte-core
 
 
 #. Verify all the pods have come up correctly
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:491]:
```suggestion
#. Verify if all the pods have come up correctly
```

File: rsts/deployment/aws/manual.rst [hunk 12]
```diff
@@ -477,9 +504,21 @@ Uninstalling Flyte
 Upgrading Flyte
 ---------------
 
-.. code-block:: bash
+   .. tabbed:: Flyte Native scheduler
+
+      * Install Flyte with flyte native scheduler:
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:509]:
```suggestion
      * Install Flyte with Flyte native scheduler
```

File: rsts/deployment/aws/manual.rst [hunk 13]
```diff
@@ -477,9 +504,21 @@ Uninstalling Flyte
...
+
+        .. code-block:: bash
+
+           helm upgrade -n flyte -f values-eks.yaml --create-namespace flyte flyteorg/flyte-core
+
+   .. tabbed:: AWS scheduler
+
+      * Install Flyte with flyte aws scheduler
```
Comment (samhita-alla) [inline rsts/deployment/aws/manual.rst:517]:
```suggestion
      * Install Flyte with Flyte AWS scheduler
```

File: rsts/deployment/aws/manual.rst [hunk 14]
```diff
@@ -446,22 +468,27 @@ Search and replace the following
...
+.. tabbed:: AWS scheduler
 
-   cd helm
-   helm install -n flyte -f values-eks.yaml --create-namespace flyte .
+  * Install Flyte with Flyte AWS Scheduler
 
+    .. code-block:: bash
 
-#. Verify all the pods have come up correctly
+       helm install -n flyte -f values-eks.yaml -f values-eks-override --create-namespace flyte flyteorg/flyte-core
```
Comment (EngHabu) [inline rsts/deployment/aws/manual.rst:488]:
```suggestion
       helm install -n flyte -f values-eks.yaml -f values-eks-override.yaml --create-namespace flyte flyteorg/flyte-core
```

**Review by EngHabu — APPROVED** [review]

**Pull request merged**